### PR TITLE
Update SDWebImage to 4.4.7 and Fix TVOS bitcode issues

### LINKED ISSUE #580: Update SDWebImage to 5.0

### PR DESCRIPTION
**Pull request by newky2k**
Updated build.cake to download 5.0.2

Fixes #580

### CODE REVIEW

**Comment by adrianknight89** [comment]
@newky2k Is this getting merged soon? Also, looks like the current release is now 5.1.0.

**Comment by newky2k** [comment]
@adrianknight89  Still needs some work and i'll need to update this to the latest version

File: iOS/SDWebImage/build.cake [hunk 1]
```diff
@@ -3,17 +3,9 @@
 
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
-var IOS_PODS = new List<string> {
-	"platform :ios, '8.0'",
-	"install! 'cocoapods', :integrate_targets => false",
-	"target 'Xamarin' do",
-	"pod 'SDWebImage', '4.4.6'",
-	"pod 'SDWebImage/MapKit', '4.4.6'",
```
Comment (mattleibow) [inline iOS/SDWebImage/build.cake:11]: Are we dropping the MapKit extensions?

Comment (SotoiGhost) [reply iOS/SDWebImage/build.cake:11]: These lines are obsolete. The podfile lines with MapKit are some lines below.

Comment (mattleibow) [reply iOS/SDWebImage/build.cake:11]: Ah, I see it now. Just had a blind moment.

**Review by Redth — APPROVED** [review]

**Pull request merged**